setup/structure.js

```diff
@@ -346,3 +346,5 @@ exports.setupStructure = function (db) {
   .then(() => createProjectsSourceData(db))
   .then(() => createWbCatalogResources(db));
 }
+
+module.exports = setupStructure;
```

Comment:
if you do this, you're exporting the `setupStructure` function as the module default and you'll have to update the `setup/index.js`file.

I'd suggest to change this to a named export:

```suggestion
module.exports = {
  setupStructure
};
```